docs/reference/commandline/ps.md

```diff
@@ -207,6 +207,17 @@ The `volume` filter shows only containers that mount a specific volume or have a
     CONTAINER ID        MOUNTS
     9c3527ed70ce        remote-volume
 
+#### Network
+
+The `network` filter shows only containers that has endpoints on the provided network name.
+
+    $docker run -d --net=net1 --name=test1 ubuntu top
+    $docker run -d --net=net2 --name=test2 ubuntu top
+
+    $docker ps --filter network=net1
```

Comment:
Actually, perhaps you can add a `bash` language hint here as well, so, don't indent the code-block, but use a "fence". Like this:

    The `network` filter shows only containers that has endpoints on the provided
    network name.

    The following filter matches all containers that are connected to a network
    with a name containing `net1`.

    ```bash
    $ docker run -d --net=net1 --name=test1 ubuntu top
    $ docker run -d --net=net2 --name=test2 ubuntu top

    $ docker ps --filter network=net1
    CONTAINER ID        IMAGE       COMMAND       CREATED             STATUS              PORTS               NAMES
    9d4893ed80fe        ubuntu      "top"         10 minutes ago      Up 10 minutes                           test1
    ```